src/cluster_asm.c

```diff
@@ -733,9 +736,7 @@ void asmCallbackOnFreeClient(client *c) {
         serverAssert(task->rdb_channel_state == ASM_RDBCHANNEL_TRANSFER);
         task->rdb_channel_conn = NULL; /* Will be freed by freeClient */
         c->flags &= ~CLIENT_MASTER;
-        asmTaskSetError(task, "Import rdb channel connection is closed, state: %s",
-                              asmTaskStateToString(task->rdb_channel_state));
-        asmImportFailed(task);
+        asmTaskSetFailed(task, "rdb channel - Connection is closed");
```

Comment:
Do you think the first letter should be capitalized? i.e. `RDB channel`, `Main channel`
P.S. don't worry test cases

Reply:
okay sure